### ISSUE TITLE
Compilation Error for 'break' keyword

### ISSUE THREAD

**Issue by Woccz** [dbaumgarten/yodk#44]
For some reason the code
```
a=1
:o=0

while ++a < :i do
    b=1
    while ++b < sqrt(a) do
        if b%a then
            o++
            break
        end
    end
end

:o += :i==2

:done = 1
```
gets compiled to 
```
b=1 :o=0
if++b>=:i thengoto4end c=1
if++c<sqrt b thenifc%b thend++ gotoendwhile2 end goto3end goto2
:o+=:i==2 :done=1 goto1
```

There should be no `while` in the compiled to yolol code.

Thank you.

**Comment by Woccz** [comment]
`break` just seems broken.

**Comment by dbaumgarten** [comment]
Good catch. That definitly doesn't look right.
"endwhile2" is an internal line-label that is generated when converting the second loop and it is supposed to be replaced with an actual line-number as one of the last steps during compilation. For some reason that is not happening here.
I will look into it 👍

**Comment by dbaumgarten** [comment]
Found the issue. When converting a loop to an inline-yolol-loop, the converter did not add the label for the end of the loop. Therefore "endwhile2" was not recognized as a goto-to-label and was not replaced by an actual number. 

The example code is now (or at least with the next published version) compiled to:
```
b=1 :o=0
if++b>=:i thengoto4end c=1
if++c<sqrt b thenifc%b thend++ goto2end goto3end goto2
:o+=:i==2 :done=1 goto1
```

**Comment by Woccz** [comment]
Great 👍

**Comment by dbaumgarten** [comment]
Published as part of v0.1.4
